File: .superpowers/brainstorm/serve.js
const http = require('http');
const fs = require('fs');
const path = require('path');

const PORT = 8765;
const DIR = 'D:\\FieldVision AI\\.superpowers\\brainstorm\\session\\content';

const MIME = { '.html': 'text/html', '.css': 'text/css', '.js': 'application/javascript' };

const server = http.createServer((req, res) => {
  const file = path.join(DIR, req.url === '/' ? 'system-architecture.html' : req.url);
  if (!fs.existsSync(file)) { res.writeHead(404); res.end('Not found'); return; }
  const ext = path.extname(file);
  res.writeHead(200, { 'Content-Type': MIME[ext] || 'text/html' });
  res.end(fs.readFileSync(file));
});

server.listen(PORT, '127.0.0.1', () => {
  console.log(`Server running at http://localhost:${PORT}`);
});

File: .superpowers/brainstorm/serve.py
import http.server
import socketserver
import os

PORT = 8765
DIRECTORY = r"D:\FieldVision AI\.superpowers\brainstorm\session\content"

os.chdir(DIRECTORY)

handler = http.server.SimpleHTTPRequestHandler
with socketserver.TCPServer(("127.0.0.1", PORT), handler) as httpd:
    print(f"Serving at http://localhost:{PORT}")
    httpd.serve_forever()
